app/templates/views/accessibility_statement.html

```diff
@@ -107,27 +107,27 @@ <h3 class="heading-small" id="non-compliance">Non-compliance with the WCAG versi
 
   <ol class="govuk-list govuk-list--number">
     <li>
-      The JAWS and NVDA screen readers do not announce the ‘copy to clipboard’ button. This fails <a class="govuk-link govuk-link--no-visited-state" href="https://www.w3.org/WAI/WCAG21/Understanding/status-messages.html">success criterion 4.1.3: status messages</a>. We plan to fix this in February 2023.
+      Some icons do not appear when using high contrast mode on Windows. This fails <a class="govuk-link govuk-link--no-visited-state" href="https://www.w3.org/WAI/WCAG21/Understanding/visual-presentation.html">success criterion 1.4.8: visual presentation</a>. We have started working on this issue. We aim to fix it in March 2023.
     </li>
     <li>
-      Some icons do not appear when using high contrast mode on Windows. This fails <a class="govuk-link govuk-link--no-visited-state" href="https://www.w3.org/WAI/WCAG21/Understanding/visual-presentation.html">success criterion 1.4.8: visual presentation</a>. We plan to fix this in February 2023.
+      The JAWS and NVDA screen readers do not announce anything when you select the ‘copy to clipboard’ button. This fails <a class="govuk-link govuk-link--no-visited-state" href="https://www.w3.org/WAI/WCAG21/Understanding/status-messages.html">success criterion 4.1.3: status messages</a>. We plan to fix this in April 2023.
     </li>
   </ol>
 
   <h3 class="heading-small" id="problems-with-using-the-interface">Problems with using the interface</h3>
 
   <ol class="govuk-list govuk-list--number">
     <li>
-      Our validation errors cause problems for screen readers. This means it can be hard to know when an error has occurred or what to do next. We plan to fix this in January 2022.
+      Our validation errors cause problems for screen readers. This means it can be hard to know when an error has occurred or what to do next. We have started working on this issue. We are still exploring how much work it will take to fix it.
     </li>
     <li>
       The JAWS screen reader announces template folder page headings as multiple sentences. We plan to fix this in March 2023.
```

Comment:
@tombye will this do as a placeholder while we do the scoping?

```suggestion
      The JAWS screen reader announces template folder page headings as multiple sentences. We are investigating this issue and plan to fix it by June 2023.
```